Sources/MongoQueue/MongoQueue.swift

```diff
@@ -679,11 +259,10 @@ public final class MongoQueue {
         
         let model = try TaskModel(representing: task)
         
-        // TODO: WriteConcern majority
-//        var writeConcern = WriteConcern()
-//        writeConcern.acknowledgement = .majority
+        var writeConcern = WriteConcern()
```

Comment:
@Joannis This depends on https://github.com/orlandos-nl/MongoKitten/pull/271

Reply:
delete & update should also get this